### ISSUE TITLE
`function-comma-newline-after` complains in a multi-line value-list

### ISSUE THREAD

**Issue by MadLittleMods** [stylelint/stylelint#462]
When using [`function-comma-newline-after`](https://github.com/stylelint/stylelint/blob/master/src/rules/function-comma-newline-after/README.md), in a whitespace-newline separated value-list, it complains about wanting the function parameters to be multiline as well.

```
"function-comma-newline-after": [2, "always-multi-line"],

Expected newline after "," in a multi-line list (function-comma-newline-after)
```

```css
.foo {
	box-shadow:
		inset 0 8px 8px -8px rgba(0, 0, 0, 1),
		inset 0 -10px 12px 0 #f00;
}
```

I expect it to not complain within value items in a value-list.

---

Note: If you do add the newlines as it suggests, you run into https://github.com/stylelint/stylelint/issues/463
